docs/doc-versions.adoc

```diff
@@ -0,0 +1,11 @@
+[[doc-versions]]
+[appendix]
+== Other documentation versions
+* https://docs.janusgraph.org/latest/index.html[Latest]
+* https://docs.janusgraph.org/0.3.0/index.html[Version 0.3.0]
+* https://docs.janusgraph.org/0.2.1/index.html[Version 0.2.1]
+* https://docs.janusgraph.org/0.2.0/index.html[Version 0.2.0]
+* https://docs.janusgraph.org/0.1.1/index.html[Version 0.1.1]
+* https://docs.janusgraph.org/0.1.0/index.html[Version 0.1.0]
+
+This page only lists versions 0.3.0 and prior. Please visit the latest version to https://docs.janusgraph.org/latest/doc-versions.html[see a complete list of docs versions].
```

Comment:
I'm not sure here. Possibly I just don't know some grammar rules, but as for me it should be vice verse. I.e. something like:
`Please visit the complete list of doc versions to see the latest version.` 
Maybe I'm wrong.

Reply:
This is a version the doc listings page. The issue is we don't plan to update the old ones so we wanted to provide a link to the latest version,`https://docs.janusgraph.org/latest/doc-versions.html`, which will always have a complete listing.

I'll be in the office tomorrow and I'll take a look at rewriting this sentence to make it easier to read.

Reply:
Sorry, my bad. I thought the link is pointing to all versions documentations. My mistake. Now I understand it.